conf/application.conf

```diff
@@ -1 +1,35 @@
 # https://www.playframework.com/documentation/latest/Configuration
+play.http.secret.key=${PLAY_SECRET_KEY}
+play.http.port=9000
+# The load balancer url is just while testing, it will be removed once we have the dns sorted
+play.filters.hosts {
+    allowed = [
+        "tdr-frontend-load-balancer-intg-1439549459.eu-west-2.elb.amazonaws.com",
+        "transfer-intg.nationalarchives.gov.uk",
+        "transfer-staging.nationalarchives.gov.uk",
+        "transfer-prod.nationalarchives.gov.uk",
```

Comment:
This seems like it should be configured differently in each environment (including localhost), rather than whitelisting everything for all environments.

Reply:
Maybe, this is unlikely to even be the final DNS names to be honest, it was mostly meant as a placeholder to remind us to sort it at some point. I can take it out if you want because we'll remember to add it later anyway because it won't work without it.

Reply:
I've added in separate files for different environments. There's no prod yet but we can add that later. Which file we need is specified in the Dockerfile when it runs based on an environment variable passed down from ECS. I know this isn't the only way to do it but this seems quite clean.